css/choices.css

```diff
@@ -368,10 +368,6 @@ html[data-color-scheme=dark] {
   background-image: none !important;
 }
 
-.choices:last-child {
-  margin-bottom: 3px;
-}
-
```

Comment:
@zoglo Removing this did not make any difference in my tests. What does it do?

Reply:
![image](https://github.com/user-attachments/assets/9efa8f31-a7e5-4dcd-bfc3-213005fdac11)
Above is with the margin and below is without.
That's literally the slight pixel to adjust the text :)

Reply:
I still don't get it. According to the dev tools, the `.choices:last-child` rule is not applied:

<img width="1030" alt="" src="https://github.com/user-attachments/assets/5b1303d1-ef51-4d88-8e0c-3065b1ce154c" />

Reply:
Okay, so I don't understand it but I sure see the effect :smiling Reverted in aae66d4b4381510a284492949583af697020a72d.

Reply:
![image](https://github.com/user-attachments/assets/3562bacc-1621-47fa-a490-d94083a50734)
Because this is the last child, hence not having any effect in this row. It's not the tl_filters but `tl_limit` and `tl_sorting` that gets the margin.

Even if in the case that the `<br>` get's removed, the last filter will be fine as it's the default margin anyways :)